src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const COST_PER_DAY = 40;
+  const DISCOUNT_50 = 50;
+  const DISCOUNT_20 = 20;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+
+  if (days >= LONG_TERM) {
+    return (COST_PER_DAY * days) - DISCOUNT_50;
+  }
+
+  if (days >= SHORT_TERM) {
+    return (COST_PER_DAY * days) - DISCOUNT_20;
+  }
+
+  return COST_PER_DAY * days;
```

Comment:
You now have `COST_PER_DAY * days` repeated three times - this can be transferred to a variable and reused.